sdk/keyvault/keyvault-keys/README.md

```diff
@@ -40,6 +40,18 @@ To quickly create the needed Key Vault resources in Azure and to receive a conne
 
 [![](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-sdk-for-js%2Fmaster%2Fsdk%2Fkeyvault%2Ftest-resources.json)
 
+To fill this form:
+
+1. If you know about ARM templates, or if you're interested to know about them, we recommend you to take some time to read through the [ARM template we are using for this button](https://github.com/Azure/azure-sdk-for-js/blob/keyvault/deploy-button-guide/sdk/keyvault/test-resources.json). You can read more about the concept of an ARM template in our [Intro to Microsoft Azure Resource Manager Templates (Video)](https://dev.to/azure/intro-to-microsoft-azure-resource-manager-templates-video-9cc).
+2. The subscription, resource group and location are required fields, they help specify where to create the KeyVault.
+3. On the "Base Name", you'll see `[resourceGroup().name]`. Remove the contents of this field and type any alphanumeric string. This string will be used to generate unique names for the resources that will be created.
```

Comment:
styling: you might want to use **Bold** font face for portal UI elements.  I've seen the patter in many azure docs.

Reply:
What would you like to make bold? Just sot that I can make sure I address your feedback properly.

Reply:
For example, in one of the linked doc

![image](https://user-images.githubusercontent.com/7583839/74481549-cad86a00-4e67-11ea-907d-1bf8be0fd023.png)

Reply:
I have used the bold markdown syntax on several words and names! Let me know if it's better now 🌞